File: benchmarks/javascript/squares.js
#!/usr/bin/env node

var n = 1000
var x = 10000

function main() {
  for (var i = 0; i < n; ++i) {
    for (var j = i; j < n; ++j) {
      if (i*i + j*j == x) {
        console.log(i, j)
      }
    }
  }
}

main()

File: benchmarks/javascript/squares.py
#!/usr/bin/env python2
"""
squares.py
"""
from __future__ import print_function

import sys

n = 1000
x = 10000

def main():
  for i in xrange(n):
    for j in xrange(i, n):
      if i*i + j*j == x:
        print(i, j)


if __name__ == '__main__':
  main()
